RocketChat/Rocket.Chat.ReactNative
# Chore: Update React Native Device Info to 8.4.8

## Proposed changes

Adds ios 13 support from the device info lib and resolves bugs related to it.

Below is a photo with the iphone 13 device, which before the lib update had the jump to bottom button cut off, as the lib didn't know if the device had notch or not.

## Issue(s)	


## How to test or reproduce

Log into the app, join a channel and scroll the messages until the jump to bottom button appears.

## Screenshots
![Screen Shot 2021-12-17 at 10 40 05](https://user-images.githubusercontent.com/18687097/146556585-85d05fb3-a33a-4572-af44-fa472368b51d.png)

## Types of changes




## Checklist



## Further comments
